fs/xfstests.py.data/README

```diff
@@ -0,0 +1,43 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
```

Comment:
Maybe you should separate what is env var from what is set in yaml file. Anyway, I don't see any clue about what goes in the yaml file.